pkg/controller/controller.go

```diff
@@ -155,18 +155,26 @@ func (c *Kleaner) Process(obj interface{}) {
 			return
 		}
 		job := t
-		// skip the job if it hasn't completed yet or has any active pods
-		if job.Status.CompletionTime.IsZero() || job.Status.Active > 0 {
```

Comment:
When I was working on this PR, I first added logging when this if was `true` and for failed jobs there were logs that they were skipped due to `job.Status.CompletionTime.isZero()`